numpy/core/code_generators/generate_ufunc_api.py

```diff
@@ -36,11 +36,14 @@
 static NPY_INLINE int
 _import_umath(void)
 {
-  PyObject *numpy = PyImport_ImportModule("numpy.core.umath");
+  PyObject *numpy = PyImport_ImportModule("numpy.core._multiarray");
   PyObject *c_api = NULL;
 
   if (numpy == NULL) {
-      PyErr_SetString(PyExc_ImportError, "numpy.core.umath failed to import");
+      numpy = PyImport_ImportModule("_multiarray");
```

Comment:
Why are we trying to import a top-level module?

Reply:
I don't have enough experience here to know if we want this method.

I'm just worried that this looks like a removed-in-py3k implicit relative import.

Reply:
removed. It was useful only to import `_multiarray` indepedent of the rest of numpy:
```
data = imp.find_module('_multiarray', ['cpython3_virt/lib/python3.5/site-packages/numpy/core'])
multiarray = imp.load_module('_multiarray', *data)
```
which I was doing for debugging the bootstrap of numpy.__init__